salesagility/SuiteCRM
# Fixed #4511 - Upgrade of TinyMCE to latest 4.9.x version

## Description



Moving tinymce to composer.

## Motivation and Context

Issue reference: #4511. 

## How To Test This

1. Set editor to TinyMCE in user profile.
2. Create an email template.
3. See that the TinyMCE editor works as expected.

## Types of changes


### Final checklist
